ll-property-information-view.html

```diff
@@ -146,9 +146,9 @@
         <!--<p>{{property.currencyCode}}</p>-->
       <!--</ll-definition-list>-->
 
-      <ll-definition-list title="LL UNIT ID:" hidden$="{{!property.unitId}}">
-        <p>{{property.unitId}}</p>
-      </ll-definition-list>
+      <!--<ll-definition-list title="LL UNIT ID:" hidden$="{{!property.unitId}}">-->
```

Comment:
can we just delete them? We can use github's history in the case we want to add it back..